superset-frontend/.eslintrc.js

```diff
@@ -101,6 +106,7 @@ module.exports = {
         'new-cap': 0,
         'no-bitwise': 0,
         'no-continue': 0,
+        'no-console': 0,
```

Comment:
I think we can set this to 1. We might even be able to set it to 2, since people can ad an eslint-ignore comment when it's needed, or better yet `import { logging } from '@superset-ui/core';` and do `logging.log/info/warn/error/table`

Reply:
I looked for a custom Logger but I didn't found one. Thanks for the `import { logging } from '@superset-ui/core'; `reference! This is very simple to fix, I think we have only one file using console directly. I'll enable again and fix it.